src/coq_rules.ml

```diff
@@ -111,6 +111,17 @@ let create_ccoq sctx ~dir =
   ; coqpp  = rr "coqpp"
   }
 
+(* get_libraries from Coq's ML dependencies *)
+let libs_of_coq_deps ~loc ~scope libs =
+  let lib_db = Scope.libs scope in
+  match Lib.DB.find_many ~loc lib_db
+          (List.concat_map ~f:Dune_file.Lib_dep.to_lib_names libs) with
+  | Ok libs ->
+    libs
+  | Error exn ->
+    Errors.fail loc "ML dependencies for Coq library %a"
+      (Exn.pp_uncaught ~backtrace:"") exn
```

Comment:
Is this an error meant to be seen by the user?

* If yes, then you should catch in report_error and print it properly there.
* If no, then you should use `Exn.code_error`.

Reply:
Oh indeed, the error should be the same here as in regular `library` stanzas, in concrete the common one is:
```
Error: Library "foo" not found.
```
The errors there are already registered in `Report_error`, so I guess that just raising is the way to go? [It seems to work fine here]